docs/product-migration-guide/src/main/asciidoc/migration-java-dependencies-proc.adoc

```diff
@@ -97,7 +110,27 @@ Replace it with the following dependency:
   <artifactId>uberfire-rest-client</artifactId>
 </dependency>
 ----
-. Update the version of the project artifact.
-. Save the `pom.xml` file.
-. Rebuild the project, using a regular Maven build.
-. If build errors or subsequent execution errors happen, resolve them using the information in xref:code-troubleshooting-ref[].  
+ifdef::PAM[]
+. If the following dependency exists:
```

Comment:
Hmmmm... Not a fan of breaking statements like this, but I see why you are doing it. Consider something like:
Check if the following dependency exists:
If so, replace....

Reply:
Yeah, I really hate this section. This was the format used in the last version of the doc, and I couldn't really think of a better way. However, on second thought, I just adjusted it according to a similar format I used elsewhere in the doc. What do you think now? Here's the preview: http://file.rdu.redhat.com/~sterobin/BXMSDOC-2391_BA/#migration-java-dependencies-proc